doc/howtos/realsense.rst

```diff
@@ -0,0 +1,34 @@
+Intel® RealSense™ cameras in IoT Reference OS Kit
+#################################################
+
+Intel® RealSense™ cameras are supported in IoT Reference OS Kit using
+``librealsense`` (https://github.com/IntelRealSense/librealsense)
```

Comment:
Could you turn the ``librealsense`` word into a hyperlink (instead of displaying the full hyperlink in brackets)?